docs/deploy/security_groups.md

```diff
@@ -0,0 +1,31 @@
+The  AWS Load Balancer Controller manages security groups of two categories: frontend and backend.
```

Comment:
Lets say "classifies the security group into two categories:" instead of "manages the security group of two categories:". The classification is logical from the controller's perspective.